### ISSUE TITLE
[Feature Request] Create launch script when installed from npm

### ISSUE THREAD

**Issue by bkanuka** [cstayyab/WALC#61]
After installing with npm I'd like to launch walc by just typing `walc` at the command line (or more realistically creating a .desktop file launcher). 

I beleive this is as simple as creating a `bin` feild in package.json but I'm not a node developer at all so I might be off.  Regardless, hopefully this is a small change.

**Comment by Zzombiee2361** [comment]
Why would you want to do that? Launching using `npm start` is supposed to be for development only. Just compile it using `npm run dist`, it will create `dist` directory with AppImage and Snap package. But it also create `linux-unpacked` folder, you can copy and rename that folder and running `walc` binary in there. As for desktop file launcher, your desktop environment usually has a menu editor. You can also symlink `walc` binary to `~/local/share/bin` if you want to run it from command line

**Comment by bkanuka** [comment]
This came up when I tried to install walc using `npm install -g https://github.com/cstayyab/WALC.git`.  I know it's not a "supported" method but I've installed some nodejs programs like this before.  For me, it removes the need for snap or appimage - neither of which I use, and it's an easy one-line for installing node programs.

It's a pretty simple change, and I figured it out.  I'll submit a pull request for review.
